api/graphql/tests/test_applications/test_application_create.py

```diff
@@ -0,0 +1,255 @@
+                "name": "Super organisation",
+                "address": {
+                    "streetAddress": "Testikatu 28",
+                    "postCode": "33540",
+                    "city": "Tampere",
+                },
+            },
+            "contactPerson": {
+                "firstName": "John",
+                "lastName": "Wayne",
+                "email": "john@test.com",
+                "phoneNumber": "123-123",
+            },
+            "applicationRoundPk": self.application.application_round.id,
+            "applicationEvents": [],
+            "status": ApplicationStatus.DRAFT,
+            "billingAddress": {
+                "streetAddress": "Laskukatu 1c",
+                "postCode": "33540",
+                "city": "Tampere",
+            },
+            "homeCityPk": self.city.id,
+        }
+
+    def get_schedule_data(self):
+        return {"day": 1, "begin": "10:00", "end": "16:30"}
+
+    def get_event_reservation_unit_data(self):
+        return {"priority": 22, "reservationUnit": self.reservation_unit.id}
+
+    def get_event_data(self, app_id=None):
+        return {
+            "name": "App event name",
+            "applicationEventSchedules": [self.get_schedule_data()],
+            "numPersons": 10,
+            "ageGroup": self.age_group.id,
+            "abilityGroup": self.ability_group.id,
+            "minDuration": "01:00:00",
+            "maxDuration": "02:00:00",
+            "application": app_id,
+            "eventsPerWeek": 2,
+            "biweekly": False,
+            "begin": datetime.date(2022, 8, 1).strftime("%Y-%m-%d"),
+            "end": datetime.date(2023, 2, 28).strftime("%Y-%m-%d"),
+            "purpose": self.purpose.id,
+            "eventReservationUnits": [self.get_event_reservation_unit_data()],
+            "status": ApplicationEventStatus.CREATED,
+        }
+
+    def test_create(self):
+        assert_that(Application.objects.count()).is_equal_to(1)
+        self.client.force_login(self.regular_joe)
+        data = self.get_application_data()
+
+        response = self.query(self.get_create_query(), input_data=data)
+        assert_that(response.status_code).is_equal_to(200)
+        content = json.loads(response.content)
+        app_data = content.get("data").get("createApplication")
+        assert_that(content.get("errors")).is_none()
+        assert_that(app_data.get("errors")).is_none()
+
+        assert_that(Application.objects.count()).is_equal_to(2)
+
+        application = Application.objects.get(id=app_data.get("pk"))
+        assert_that(application).is_not_none()
+        assert_that(application.status).is_equal_to(data["status"])
+
+    def test_application_create_organization_identifier_not_required(self):
+        assert_that(Application.objects.count()).is_equal_to(1)
+        self.client.force_login(self.regular_joe)
+
+        data = self.get_application_data()
+        _ = data["organisation"].pop("identifier")
```

Comment:
Is there a reason to assign the popped value to `_`?

Reply:
absolutely no reason - _ is a throwaway variable. Using it in popping is useless but for e.g when getting something which is tuple it is a convention used for _not used variables like:
`a, _, c, _ = some_func_returns_four_var_tuple()` or `for k, _ in dictionary.items():`

I'll remove it here

Reply:
removed.